api/resolvers/search.js

```diff
@@ -282,7 +288,7 @@ export default {
             fields: ['title^100', 'text'],
             fuzziness: 'AUTO',
             prefix_length: 3,
-            minimum_should_match: '60%'
+            minimum_should_match: (isUrlSearch) ? 1 : '60%'
```

Comment:
If we are exclusively searching for an url, we just need to match either the root domain or the fqdn.